log/term/terminal_windows_test.go

```diff
@@ -0,0 +1,37 @@
+package term
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/sys/windows"
+)
+
+// +build windows
+
+type myWriter struct {
+	fd uintptr
+}
+
+func (w *myWriter) Write(p []byte) (int, error) {
+	return 0, fmt.Errorf("not implemented")
+}
+
+func (w *myWriter) Fd() uintptr {
+	return w.fd
+}
+
+func TestIsTerminal(t *testing.T) {
+	handle, err := windows.GetStdHandle(windows.STD_OUTPUT_HANDLE)
```

Comment:
The test directly fetches STDOUT using `GetStdHandle` because `go test` redirects its output (and so fails) but **only** if not run with `-v` which makes for a surprising behavior otherwise !